packaging/rpm/esbulk.spec

```diff
@@ -50,6 +50,10 @@ rm -rf %{_topdir}/BUILD/%{name}
 
 %changelog
 
+* Mon Oct 2 2020 Martin Czygan
```

Comment:
Hey, are you my alter ego? I have not been strict about keep up the spec changelog, but if you do, just put your handle in or leave it out completely.